Dockerfile

```diff
@@ -11,8 +11,8 @@ COPY . ./
 RUN apt-get update && \
   apt-get install -y --no-install-recommends gcc python3-dev libssl-dev git && \
   python -m pip install --no-cache-dir --upgrade pip && \
-  python -m pip install --no-cache-dir . && \
-  python -m pip install --no-cache-dir -r requirements-hyperdrivepy.txt && \
+  python -m pip install --no-cache-dir uv && \
+  python -m uv pip install --no-cache-dir agent0@.[all] && \
```

Comment:
Adding this bloats the docker container to also contain e.g., pytorch

Reply:
agreed, so we should just do [dev] then?

Reply:
IIRC we just did no dependencies and it worked fine, so:
```
python -m uv pip install --no-cache-dir agent0@. &&
```
I did look over this with the uv install PR and purposefully left this as pip, to also avoid having uv in the image as well, but I didn't look too deeply on how big uv was.

Reply:
uv is huge (~25mb I think); so I'd rather not have it... ok I'll revert this to the original command. We should think about building in a test for validating our INSTALL instructions though.